<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpClientTestClass</class>
 <widget class="QMainWindow" name="TcpClientTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TcpClientTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLineEdit" name="lineEdit_send"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_ip">
          <property name="text">
           <string>IP:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_port">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_ip"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_port"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="ptn_connect">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 255, 0);</string>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ptn_disconnect">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 255, 0);</string>
          </property>
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTextEdit" name="textEdit_receive"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="ptn_send">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 255, 0);</string>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TcpClientTest.qrc"/>
 </resources>
 <connections/>
</ui>
